Fusor/js/observer.js

```diff
@@ -99,27 +98,60 @@ function emergency_stop() {
     }
 }
 
-
 function loadServerLog(input) {
     loadLog(input.text, true);
 }
 
 
 function loadLog(fileName, addPrefix) {
-
     stopStatus();
-    console.log("loading log: " + fileName);
```

Comment:
Please still include a console log message about the file to load, though you will need to move it down to after your file picker.

Reply:
Done, see [this line of code](https://github.com/EastsidePreparatorySchool/EastsidePreparatorySchool.github.io/pull/1/files#diff-7eb466edfb3ce594544549d90d598ba9a2bc6737cf3f6c218d5e4968ec2f51fdR144)